MANIFEST.in

```diff
@@ -1,3 +1,2 @@
-include pyproject.toml
-include *.md
-include LICENSE
+include CONTRIBUTING.md Makefile bin/*.py requirements/*
```

Comment:
Did you intentionally not mention LICENSE, pyproject.toml and README.md here?

Reply:
Yes, [based on the default inclusion](https://packaging.python.org/guides/using-manifest-in/).  That also explains the build backends' version specification below.